cli/src/commands/mint-to/index.ts

```diff
@@ -0,0 +1,80 @@
+import { Args, Command, Flags } from "@oclif/core";
+import {
+  CustomLoader,
+  getPayer,
+  getRpc,
+  getSolanaRpcUrl,
+} from "../../utils/utils";
+import { Connection, Keypair, PublicKey } from "@solana/web3.js";
+import { mintTo } from "@lightprotocol/compressed-token";
+import { confirmTx } from "@lightprotocol/stateless.js";
+
+class MintToCommand extends Command {
+  // light mint-to --mint PublicKey --to PublicKey --amount <u64>
+  static summary = "Mint tokens to an account.";
+
+  static examples = [
+    "$ light mint-to --mint PublicKey --to PublicKey --amount 1000",
+  ];
+
+  static flags = {
+    // ...standardFlags,
+    // ...confirmOptionsFlags,
+    mint: Flags.string({
+      description: "Mint.",
+      required: false,
+    }),
+    to: Flags.string({
+      description: "Recipient.",
+      required: true,
+    }),
+    amount: Flags.integer({
+      description: "Amount.",
+      required: true,
+    }),
+  };
+
+  static args = {};
+
+  async run() {
+    const { args, flags } = await this.parse(MintToCommand);
+    const mint = flags["mint"];
+    const to = flags["to"];
+    const amount = flags["amount"];
+    if (!to || !amount) {
+      throw new Error("Invalid arguments");
+    }
+
+    const loader = new CustomLoader(`Performing mint-to...\n`);
+    loader.start();
+
+    try {
+      const mintPublicKey = mint
+        ? new PublicKey(mint)
+        : Keypair.generate().publicKey;
+      const toPublicKey = new PublicKey(to);
+      const payer = getPayer();
+      const connection = new Connection(getSolanaRpcUrl());
+
+      // TODO: remove this, it's just for testing
+      const sig = await connection.requestAirdrop(payer.publicKey, 2e9);
+      await confirmTx(connection, sig);
+
+      const txId = await mintTo(
+        connection,
+        payer,
+        mintPublicKey,
+        toPublicKey,
+        payer.publicKey,
```

Comment:
note that this should be the mintAuthority - so users will always have to pass this into the cmd.
(i think by default for the equivalent cmd in the solana cli, they read from the default solana keypair (i.e. by default they assume that payer and mintAuthority are the same (which they usually are but that depends on what you pass as authority into createMint